src/mango/src/mango_idx_view.erl

```diff
@@ -68,13 +69,14 @@ remove(#doc{body={Props0}}=DDoc, Idx) ->
     if Views2 /= Views1 -> ok; true ->
         ?MANGO_ERROR({index_not_found, Idx#idx.name})
     end,
-    Props1 = case Views2 of
+    Props = case Views2 of
         [] ->
-            lists:keydelete(<<"views">>, 1, Props0);
+            Props1 = lists:keydelete(<<"views">>, 1, Props0),
+            lists:keydelete(<<"autoupdate">>, 1, Props1);
```

Comment:
Removing autoupdate here seems to suggest that we're worried about it existing. Given that this only runs rarely its not going to provide much value. Plus it makes me concerned that we need to be a lot more specific on how query design docs are written if it is in fact important that autoupdate be disabled (rather than being slightly wasteful).

Reply:
Can you link to that check? I'm not able to find it. I'm guessing that was some sort of check to try and avoid deleting design docs that had been edited by hand or similar?